src/meshes/triangle_mesh.template.h

```diff
@@ -2491,6 +2528,18 @@ namespace oomph
       return Min_permitted_angle;
     }
 
+    /// Read-only access to the min element interior angle for adaption
+    const double& min_element_interior_angle_for_adaption() const
+    {
+      return this->Min_element_interior_angle_for_adaption;
+    }
+
+    /// Set the min element interior angle for adaption
+    void set_min_element_interior_angle_for_adaption(const double& new_angle)
```

Comment:
Definitely think that using "after" rather than "for" conveys the sense better for this function. Presumably this can be used at any point between adaptations as a means to change the minimum angle.

Reply:
TBH, I still don't really understand why we can't just include this parameter into the arguments class that passes all the other parameters (with sensible defaults for those that aren't explicitly specified) to the constructor. Distinguishing between what happens when the mesh is first generated and during the next adaptation seems a bit odd (and implies that it's a hacky work-around for not being able to set the parameter straightaway. Why would I want to do the first calculation on a mesh that I don't like long term?

Reply:
I am not distinguishing between first generated mesh and the subsequent adaption steps. If you want to, you can use the set_min_element_interior_angle(new_angle).

There are three constructors for Triangle meshes: a default one with no arguments, one with our lovely arguments class `TriangleMeshParameters` and one specifying a node file name, element file name and poly file name.
The parameters class is not stored within `TriangleMesh` so the min interior angle would be lost after constructor unless we store it within the class.
We need to pass it as an optional argument in the other cases as the mesh gets built in the constructor, so we need this parameter in the constructor.